p2p/net/multistream-muxer/multistream.go

```diff
@@ -0,0 +1,75 @@
+// Package multistream implements a peerstream transport using
```

Comment:
File name: `muxer-multistream`? It would be nice if this matched the security version.

Location: Maybe put this in `/muxer`? it's a muxer combinator, but it's also a muxer. But I don't feel strongly about that.

Package name: Not multistream (going to get a bit confusing?). But I'm not sure what to call it. Maybe `multistream_muxer`?

Reply:
Not a big fan, but let's go with muxer-multistream for now.

Reply:
Yeah, all these names are long and annoying. I just want to be as consistent as possible.

Reply:
Is there anything else I should change? It's muxer-multistream everywhere now.